deploy/k8s/worker-sgx.yml

```diff
@@ -0,0 +1,62 @@
+---
+
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: faasm-worker
+  namespace: faasm
+spec:
+  template:
+    metadata:
+      labels:
+        app: faasm-worker
+    spec:
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+            - podAffinityTerm:
+                labelSelector:
+                  matchLabels:
+                    app: faasm-worker
+                topologyKey: kubernetes.io/hostname
+              weight: 100
+
+      containers:
+        - image: faasm/worker-sgx:0.8.3
+          resources:
```

Comment:
We need to add this `resources` field to make sure the SGX device is available in the pods ([see azure's docs](https://docs.microsoft.com/en-us/azure/confidential-computing/confidential-nodes-aks-overview)).